src/views/offering/AddComputeOffering.vue

```diff
@@ -333,6 +333,28 @@
             }]"
             :placeholder="this.$t('label.networkrate')"/>
         </a-form-item>
+          <a-form-item>
+            <span slot="label">
+              {{ $t('label.root.disk.size') }}
+              <a-tooltip :title="apiParams.provisioningtype.description">
```

Comment:
```suggestion
              <a-tooltip :title="apiParams.rootdisksize.description">
```

Reply:
Thanks for the review @davidjumani, fixed.